app/frontend/src/store/index.js

```diff
@@ -1,10 +1,12 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 
+import form from '@/store/modules/minesAttestations/minesAttestationsForm.js';
+
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-  modules: {},
+  modules: { form },
```

Comment:
We should strongly consider using lazy loading this time around for the form - have it be mounted on Root.vue and unmounted upon destroy.

Reply:
Temporary. Definitely planning on lazy loading it.